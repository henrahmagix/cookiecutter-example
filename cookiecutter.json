{
    "folder_name": "myfolder",
    "file_name": "myfilename",
    "file_extension": "myext",
    "file_contents": "My file contents",
    "array_of_options": ["a", "b", "c"],
    "object_of_options": {
        "a": "The letter A",
        "b": "The letter B",
        "c": "The letter C"
    },
    "_copy_without_render": [
        "{{cookiecutter.next_*}}"
    ]
}
