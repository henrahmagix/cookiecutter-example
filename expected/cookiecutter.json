{
    "next_folder_name": "nextfolder",
    "next_contents": "The next contents"
}
